force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -226,9 +277,9 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Double complexOrderOfOperations(Integer a, Integer b, Integer c, Integer d, Integer e) {
         // Add parentheses around the multiplication and subtraction operations so that they are performed before division and addition
-        Double result = a * b - c / (Double) d + e;
+        Double result = ((a * b - c) / (d)) + e;
```

Comment:
Again, you can leave out one set of parentheses since it isn't technically needed due to the order of operations. You would also want to keep the `(Double)` in since this is casting d to a Double (this ensures that the final result is a Double although it appears it's not always necessary -- so don't worry about that for now).
```suggestion
        Double result = (a * b - c) / (Double) d + e;
```
In the case above, (a * b - c) would be evaluated first, then divided by d, then added to e.